bin/install_travis.sh

```diff
@@ -1,37 +1,82 @@
 #!/usr/bin/env bash
 
-sudo apt-get update
-sudo apt-get install cmake libgmp-dev
+if [[ "${CC}" == "" ]]; then
+    export CC=gcc
+    export CXX=g++
+fi
+if [[ "${TRAVIS_OS_NAME}" == "osx" ]] && [[ "${CC}" == "gcc" ]]; then
+    export CC=gcc-4.8
+    export CXX=g++-4.8
```

Comment:
Cannot you just say `gcc` and `g++`? Why do you need to specify a particular version? If Travis updates, we'll have to change this.